services/cognito/app/controllers/pools_controller.rb

```diff
@@ -1,4 +1,20 @@
 # frozen_string_literal: true
 
 class PoolsController < Cognito::ApplicationController
+  def create
+    res = PoolCreate.call(params: create_params, user: context[:user])
```

Comment:
We will need to come up with a clean pattern for this. We start to have many variances of this.